src/hotspot/share/cds/cdsConstants.cpp

```diff
@@ -31,18 +31,18 @@
 #include "utilities/globalDefinitions.hpp"
 
 CDSConst CDSConstants::offsets[] = {
-  { "GenericCDSFileMapHeader::_magic",                     offset_of(GenericCDSFileMapHeader, _magic)          },
-  { "GenericCDSFileMapHeader::_crc",                       offset_of(GenericCDSFileMapHeader, _crc)            },
-  { "GenericCDSFileMapHeader::_version",                   offset_of(GenericCDSFileMapHeader, _version)        },
-  { "GenericCDSFileMapHeader::_header_size",               offset_of(GenericCDSFileMapHeader, _header_size)    },
-  { "GenericCDSFileMapHeader::_common_app_classpath_prefix_size", offset_of(GenericCDSFileMapHeader, _common_app_classpath_prefix_size) },
+  { "GenericCDSFileMapHeader::_magic",                     offset_of(GenericCDSFileMapHeader, _magic)                    },
+  { "GenericCDSFileMapHeader::_crc",                       offset_of(GenericCDSFileMapHeader, _crc)                      },
+  { "GenericCDSFileMapHeader::_version",                   offset_of(GenericCDSFileMapHeader, _version)                  },
+  { "GenericCDSFileMapHeader::_header_size",               offset_of(GenericCDSFileMapHeader, _header_size)              },
   { "GenericCDSFileMapHeader::_base_archive_name_offset",  offset_of(GenericCDSFileMapHeader, _base_archive_name_offset) },
```

Comment:
I think you can delete one space between "," and "offset_of" and also adjust other lines accordingly.